core/domain/skill_services_test.py

```diff
@@ -1068,11 +1123,34 @@ def test_migrate_skill_contents_to_latest_schema(self):
 
         self.assertEqual(skill.skill_contents_schema_version, 2)
 
+        self.assertEqual(
+            (skill.skill_contents.to_dict())['explanation']['html'],
```

Comment:
Any specific reason for conversion to dict? (here and elsewhere)

Reply:
changed, I've kept to_dict() only where needed.